README.md

```diff
@@ -79,26 +79,72 @@ _Diagram showing an example layout of group-related feeds. Note that the shards
 use-case will likely not be those shown, see how they are determined below._
 
 
-### The additions feed
+### 1. The additions feed
+
+The purpose of this feed is to hold messages for coordination of joining groups.
+
+- 1.1 Each peer running this spec MUST have an invitations feed
+- 1.2 Each peer MUST deterministically place their invitations feed as a subfeed, such that:
+  - their invitations feed is a direct subfeed of a shard feed
+  - the shard feed is a subfeed of the v1 subfeed, and derived from the string `"group/additions"` according to the v1 tree structure specified in [ssb-meta-feeds-spec].
+  - the `metafeeds/add/derived` message announcing the additions feed
+      - MUST have `feedpurpose` equal to `"group/additions"`,
+      - MUST be of feed format `classic`
+      - MUST have `metadata` equal to `{ directMessage: publicKey }` where
```

Comment:
This `metadata` needs to follow https://github.com/ssbc/ssb-meta-feeds-dm-spec/pull/3